src/components/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+export const HomePage = () => {
+  return (
+    <h1 className="title">Home Page</h1>
```

Comment:
if you don't have any logic then you can return jsx without usinf word `return` , just in round brackets
` () => (`

(if linter allows)

check in all components

Reply:
every time in new file I just type shortcut 'rfa' to faster create new component with VS Code help...